config.example.json

```diff
@@ -17,5 +17,10 @@
   "ntfy_topic": "washing-machine",
   "ntfy_timeout_s": 10,
   "ntfy_tags": "alarm_clock",
-  "ntfy_priority": 3
+  "ntfy_priority": 3,
+  "heartbeat_url": "",
+  "heartbeat_interval_s": 60,
+  "heartbeat_liveness_threshold_s": 120,
```

Comment:
`heartbeat_interval_s` and `heartbeat_liveness_threshold_s` don't need to be configurable, since the internal ticker is hardcoded at 1 minute.